openedx_external_enrollments/api/v0/views.py

```diff
@@ -97,20 +98,26 @@ def post(self, request):
         """
         try:
             # Getting the corresponding enrollment controller
-            SalesforceEnrollment()
+            salesforce_controller = SalesforceEnrollment()
         except Exception:  # pylint: disable=broad-except
             LOG.info("Can't instantiate Salesforce enrollment controller")
             return JsonResponse(
                 {"info": "Can't instantiate Salesforce enrollment controller"},
                 status=status.HTTP_200_OK,
             )
         else:
+            if str(salesforce_controller) not in configuration_helpers.get_value("VALID_EXTERNAL_TARGETS", []):
+                LOG.info("Salesforce enrollment controller not enabled for this site")
+                return JsonResponse(
+                    {"info": "Salesforce enrollment controller not enabled for this site"},
```

Comment:
nit: Missing period at the end of the text. @diegomillan If you want, you can create a variable for this text since it's used twice.